src/main/OP.ts

```diff
@@ -168,8 +175,10 @@ function createVerifyRequestOptsFromBuilderOrExistingOpts(opts: { builder?: OPBu
         verification: {
           mode: VerificationMode.INTERNAL,
           resolveOpts: {
-            didMethods: opts.builder.didMethods,
-            resolver: opts.builder.resolver ? getResolver({ resolver: opts.builder.resolver }) : getResolver({ didMethods: opts.builder.didMethods }),
+            didMethods: opts.builder.subjectSyntaxTypesSupported,
```

Comment:
We need to do filtering on did: in here, and add a todo for support of other subjectSyntaxTypes

Reply:
added filter and a ticket https://sphereon.atlassian.net/browse/VDX-126